root/static/scripts/edit/externalLinks.js

```diff
@@ -577,8 +577,8 @@ MB.createExternalLinksEditor = function (options: InitialOptionsT) {
     let match;
 
     while (match = seededLinkRegex.exec(window.location.search)) {
-      (urls[match[1]] = urls[match[1]] || {})[match[2]] =
-        decodeURIComponent(match[3]);
+      const [, index, key, value] = match;
```

Comment:
An inline comment within `[,` would clarify that `match[0]` is not used at purpose.

Reply:
Changed, let me know if it looks ok

Reply:
it looks ok, thanks